react/src/App.jsx

```diff
@@ -0,0 +1,302 @@
+import { useState, useCallback, useEffect } from "react";
+
+function App() {
+  const [position, setPosition] = useState("bottom-right");
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const [selectedPrivacyOption, setSelectedPrivacyOption] = useState("disable");
+  const [selectedFullScreenOption, setSelectedFullScreenOption] =
+    useState("exitFullScreen");
+
+  const [selectedLauncherOption, setSelectedLauncherOption] =
+    useState("showLauncher");
+
+  const [unreadMessageCount, setMessageUnreadCount] = useState(0);
+  useState(false);
+
+  const [message, setMessage] = useState("");
+  const [messageEventListenerIsAdded, setmessageEventListenerIsAdded] =
+    useState(false);
+
+  const onUpdateHelpshiftConfig = () => {
+    window.Helpshift("updateHelpshiftConfig");
+  };
+
+  useEffect(() => {
+    const newInterval = setInterval(() => {
+      setMessage("");
+    }, 3000);
+
+    return () => clearInterval(newInterval);
+  }, [message]);
+
+  const onHandleFullPrivacyChange = (e) => {
+    const val = e.target.value;
+    setSelectedPrivacyOption(val);
+    if (val === "enable") {
+      window.helpshiftConfig.fullPrivacy = true;
+    } else {
+      window.helpshiftConfig.fullPrivacy = false;
+    }
+    onUpdateHelpshiftConfig();
```

Comment:
Here, and in all other places, directly call 
`window.Helpshift("updateHelpshiftConfig");`